internal/store/organization_invitations.go

```diff
@@ -104,3 +108,17 @@ func (s *Store) UpdateOrganizationInvitation(orgID, invitationID string, invitat
 		return nil, fmt.Errorf("%w: %s", errUnsupportedService, s.service)
 	}
 }
+
+// InviteUser encapsulate the logic to manage different cloud providers.
+func (s *Store) InviteUser(orgID string, invitation *atlas.Invitation) (*atlas.Invitation, error) {
+	switch s.service {
+	case config.CloudService:
+		result, _, err := s.client.(*atlas.Client).Organizations.InviteUser(context.Background(), orgID, invitation)
+		return result, err
+	case config.CloudManagerService, config.OpsManagerService:
+		result, _, err := s.client.(*opsmngr.Client).Organizations.InviteUser(context.Background(), orgID, invitation)
+		return result, err
+	default:
```

Comment:
[q] Just double checking, we don't intend to include the ops manager service? (I see you're using the om client for CM, which is why I'm curious)

Reply:
no, we are not going to include ops manager commands for invitation management as part of this epic for now but we may change idea if there is time left once I finish the epic